### PR TITLE
block AWS cronjob

### PR DESCRIPTION
**Pull request by metadave**

### CODE REVIEW

File: k8s/tools/block-aws/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+FROM python:2-slim
```
Comment (jgmize) [inline k8s/tools/block-aws/Dockerfile]: Have you tried this with python 3?

Comment (metadave) [reply k8s/tools/block-aws/Dockerfile]: it was supposed to be python3! Great catch!

**Comment by metadave** [comment]
- https://github.com/kubernetes-client/python-base/blob/789de6a60dea9442ee47601a788d92c653ea69e9/config/incluster_config.py
- https://github.com/kubernetes-client/python-base/blob/789de6a60dea9442ee47601a788d92c653ea69e9/config/incluster_config.py

File: k8s/tools/block-aws/build.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+GIT_COMMIT_RAW=$(git rev-parse HEAD)
+GIT_COMMIT=${GIT_COMMIT_RAW:0:7}
```
Comment (jgmize) [inline k8s/tools/block-aws/build.sh]: This can be simplified to
`GIT_COMMIT=$(git rev-parse --short HEAD)`

**Comment by metadave** [comment]
TODO: add DMS + secret

**Pull request merged**